internal/leafnodes/shared_runner_test.go

```diff
@@ -96,6 +96,24 @@ func SynchronousSharedRunnerBehaviors(build func(body interface{}, timeout time.
 				Ω(failure.ForwardedPanic).Should(Equal("ack!"))
 			})
 		})
+
+		Context("when a panic occurs with a nil value", func() {
+			BeforeEach(func() {
+				outcome, failure = build(func() {
+					didRun = true
+					innerCodeLocation = codelocation.New(0)
+					panic(nil)
+				}, 0, failer, componentCodeLocation).Run()
+			})
+
+			It("should return the nil-valued panic", func() {
+				Ω(didRun).Should(BeTrue())
+
+				Ω(outcome).Should(Equal(types.SpecStatePanicked))
+				Ω(failure.ForwardedPanic).Should(Equal("<nil>"))
```

Comment:
I went back and forth on whether this message should be more verbose. On one hand, `<nil>` is consistent with how Ginkgo normally behaves when you panic (and as you mentioned, it indicates that test panicked along with the `nil` value). On the other hand, you'll get this same message when you call `runtime.Goexit()`, and that could be confusing.